WalletWasabi/WebClients/BlockstreamInfo/BlockstreamInfoClient.cs

```diff
@@ -0,0 +1,88 @@
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using WalletWasabi.Blockchain.Mempool;
+using WalletWasabi.Tor.Http.Extensions;
+
+namespace WalletWasabi.WebClients.BlockstreamInfo
+{
+	public class BlockstreamInfoClient : IMempoolSyncer
+	{
+		private bool _disposedValue;
+
+		public BlockstreamInfoClient(Network network)
+		{
+			var baseUrl = "https://blockstream.info/";
```

Comment:
```suggestion
			var baseUrl = "http://explorerzydxu5ecjrkwceayqybizmpjjznk5izmitf2modhcusuqlid.onion/";
```

Reply:
It's the server. Reliabilty is important.

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/5118#issuecomment-771679008